# an object view in the original sensor frame
sensor_msgs/PointCloud2 view

# transform to register the view w.r.t. some frame
geometry_msgs/TransformStamped object_frame_transform
